rpc/src/dialect/v0_34.rs

```diff
@@ -44,15 +44,17 @@ pub struct EventAttribute {
     /// The event key.
     #[serde(
         serialize_with = "base64string::serialize",
-        deserialize_with = "base64string::deserialize_to_string"
+        deserialize_with = "base64string::deserialize"
     )]
-    pub key: String,
+    pub key: Vec<u8>,
```

Comment:
While in practice I suspect all event keys are valid UTF-8 strings encoded as base64, since the Protobuf schema does not enforce this, I believe it warrants changing its type to `Vec<u8>` as well.